server/services/conditionsService.js

```diff
@@ -8,12 +8,18 @@ module.exports = function createConditionsService(licenceClient) {
         }
     }
 
-    async function getAdditionalConditions() {
+    async function getAdditionalConditions(licence = null) {
         try {
             const conditions = await licenceClient.getAdditionalConditions();
 
+            if (licence && licence.additionalConditions) {
```

Comment:
Using ramda `path` would remove the need of a default value `licence = null` as well as simplify your if statment to `if (additionalConditions) ...`
But that's a just another way of writting thid